scripts/dev/generate_config_types.py

```diff
@@ -0,0 +1,236 @@
+"""This script auto-generates the `qutebrowser/config/configcontainer.py` file."""
+
+from __future__ import annotations
```

Comment:
looks like there's a lint rule against `from __future__ import annotations`? What issues does this cause? I'm not aware of any.